skin.css

```diff
@@ -4,10 +4,39 @@
 
 button.btn {
   border-radius: 3px;
-  border-style: solid;
-  border-width: 1px;
+  border: 1px #CECECE solid;
   font-weight: normal;
   line-height: 1.15;
   text-align: center;
   vertical-align: middle;
+  padding: 13 58;

```

Comment:
I would add `px` to be explicit about which unit this is using.

Reply:
Thanks! my bad I forgot to add px, I will add.